### ISSUE TITLE
Soft ban after 10 pokestops

### ISSUE THREAD

**Issue by asyven** [AnthonyOSX/GO-Bot#18]
Can u add new feature: 
Repeat farming 10pokestops around me (pos) every 5min?

sry 4 my engl

**Comment by jdspringr07** [comment]
Did you try to login to the app? When you made the account, did you login to it first at your location, then start the bot shortly thereafter at another? 

I haven't had a softban on a single account until I open it up too soon after closing the bot in another location.

**Comment by asyven** [comment]
Only use bot at location around me

**Comment by jdspringr07** [comment]
Yeah, but u gotta realize that "around you" can mean quite a significant distance from where you're at. If you are 3km away on the bot and immediately try to login, you will get a softban, lol.

**Comment by TaylorZaneKirk** [comment]
Softbans typically only last around 30 minutes. Is this correct in most cases that you are aware of?

**Comment by jdspringr07** [comment]
Most times, i can access again within 10 minutes. Only time ive been soft banned is when i was trying to force it (i.e. shutdown bot 30 miles away and immediately open up app at my location)

**Comment by lazerplain1** [comment]
the bot loops around in the same spot you didnt get banned
